relay-server/src/services/processor.rs

```diff
@@ -99,7 +99,7 @@ mod span;
 mod transaction;
 pub use span::extract_transaction_span;
 
-#[cfg(feature = "processing")]
+#[cfg(sentry)]
```

Comment:
I assume this needs to be:

```suggestion
#[cfg(all(sentry, feature = "processing"))]
```

You can build with sentry but without processing.

Reply:
Yeah that will need to change at a couple of spots.